core/error.go

```diff
@@ -38,17 +37,12 @@ type Error struct {
 // Error() allows *core.Error to present the `error` interface.
 func (e *Error) Error() string {
 	var ret string
+	ret = e.desc + "\n"
 
-	if os.Getenv("CONTIV_TRACE") != "" {
-		ret = e.desc + "\n"
-
-		for _, stack := range e.stack {
-			ret += fmt.Sprintf("%s [%s %d]\n", stack.fun, stack.file, stack.line)
-		}
-	} else {
-		ret = fmt.Sprintf("%s [%s %s %d]", e.desc, e.stack[0].fun, e.stack[0].file, e.stack[0].line)
+	// TODO: use github.com/pkg/errors
+	for _, stack := range e.stack {
+		ret += fmt.Sprintf("%s [%s %d]\n", stack.fun, stack.file, stack.line)
```

Comment:
kinda feelin this is a diff PR

Reply:
i'm trying to get rid of `CONTIV_TRACE`, thats why only removed the condition part, yeah, the core.error needs to be retired

Reply:
ok